tests/mgbench/client.cpp

```diff
@@ -158,6 +188,34 @@ class Metadata final {
   std::map<std::string, Record> storage_;
 };
 
+nlohmann::json LatencyStatistics(std::vector<std::vector<double>> &worker_query_latency) {
+  nlohmann::json statistics = nlohmann::json::object();
+  std::vector<double> query_latency;
+  for (int i = 0; i < FLAGS_num_workers; i++) {
+    for (auto &e : worker_query_latency[i]) {
+      query_latency.push_back(e);
+    }
+  }
+  auto iterations = query_latency.size();
+  if (iterations > 10) {
```

Comment:
Instead of a new variable, just use `query_latency.size()` + `10` should be a constant or runtime flag value?

Reply:
It sould not be a runtime flag, since number of iterations is defined by the input number of queries. This was different in benchmark.py where we where executing some query defined number of iterations. 
This just defines a lower bound after which there is no sense to calculate latency values. Will transform this to local const varabile since it scope should be local.